packages/examples/component-library/src/atoms/Logo/Logo.stories.tsx

```diff
@@ -1,15 +1,18 @@
-import Readme from "./README.md";
-
-import { storiesOf } from "@storybook/react";
 import React from "react";
+
 import Logo from "./Logo";
 
-storiesOf("Logo", module).add(
-    "Default",
-    () => {
-        return <Logo speed={"30s"} />;
-    },
-    {
-        notes: { markdown: Readme },
-    },
-);
+interface LogoStoryProps {
+    speed: string;
```

Comment:
Is this how to set up knobs now?

Reply:
No but then again kinda. Knobs are for the Canvas view. This stuff is more for the Document view. They have a lot of overlap  and in general it does seem like the Document view is really powerful but my guess is there are situations where you might need more power and that's where the canvas/knobs come in.